### PR TITLE
(DOCSP-27346): Intros & Prereqs for MongoDB Data Access pages

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Add standardized introductions and 'Prerequisites' sections for SDKs' MongoDB Data Access pages

> **Note**
>
> Must be merged after https://github.com/mongodb/docs-app-services/pull/304. This PR is dependent on a ref created in that PR.

### Jira

- https://jira.mongodb.org/browse/DOCSP-27346

### Staged Changes

- [Query MongoDB (.NET)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-27346/sdk/dotnet/app-services/mongodb-remote-access/)
- [Query MongoDB (Java)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-27346/sdk/java/examples/mongodb-remote-access/)
- [Query MongoDB (Node.js)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-27346/sdk/node/examples/query-mongodb/)
- [Query MongoDB (React Native)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-27346/sdk/react-native/app-services/query-mongodb/)
- [Query MongoDB (Swift)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-27346/sdk/swift/app-services/mongodb-remote-access/)
- [Query MongoDB (Web)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-27346/web/mongodb/)
### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/dotnet/app-services/mongodb-remote-access.txt [hunk 1]
```diff
@@ -10,8 +10,15 @@ Query MongoDB - .NET SDK
...
+You can query data stored in MongoDB Atlas directly from your .NET application code
+by using the Realm .NET SDK's :dotnet-sdk:`MongoClient <reference/Realms.Sync.MongoClient.html>`
+with the :manual:`Query API </tutorial/query-documents>`.
+Atlas App Services provides data access :ref:`rules <mongodb-rules>` on
+collections to securely retrieve results based on the logged-in user or the
+content of each document.
+
 The following actions enable access to a linked MongoDB Atlas cluster
-from a .NET application using the Realm .NET SDK.
+from a .NET application using the Realm .NET SDK and App Service's MongoDB Data Access.
```
Comment (mongodben) [inline source/sdk/dotnet/app-services/mongodb-remote-access.txt:21]:
```suggestion
from a .NET application using the Realm .NET SDK.
```

File: source/sdk/java/examples/mongodb-remote-access.txt [hunk 2]
```diff
@@ -10,8 +10,14 @@ Query MongoDB - Java SDK
...
 
+You can query data stored in MongoDB Atlas directly from your Android application code
+by using the Realm Java SDK with the :manual:`Query API </tutorial/query-documents>`.
+Atlas App Services provides data access :ref:`rules <mongodb-rules>` on
+collections to securely retrieve results based on the logged-in user or the
+content of each document.
+
 The following actions enable access to a linked MongoDB Atlas cluster
-from an Android application using the Realm SDK.
+from an Android application using the Realm SDK and App Service's MongoDB Data Access.
```
Comment (mongodben) [inline source/sdk/java/examples/mongodb-remote-access.txt:20]:
```suggestion
from an Android application using the Realm SDK.
```

File: source/sdk/java/examples/mongodb-remote-access.txt [hunk 3]
```diff
@@ -10,6 +10,12 @@ Query MongoDB - Java SDK
    :depth: 2
    :class: singlecol
 
+You can query data stored in MongoDB Atlas directly from your Android application code
+by using the Realm Java SDK with the :manual:`Query API </tutorial/query-documents>`.
```
Comment (dacharyc) [inline source/sdk/java/examples/mongodb-remote-access.txt:14]: Looks like the Java SDK has a MongoClient, too - we should probably link to it here: https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-27346/sdk/java/api/io/realm/mongodb/mongo/MongoClient/

The ref seems to be: `io_realm_mongodb_mongo_MongoClient`

File: source/sdk/react-native/app-services/query-mongodb.txt [hunk 4]
```diff
@@ -36,6 +43,15 @@ possible. You might want to query MongoDB when:
 While not exhaustive, these are some common use cases for querying MongoDB 
 directly.
 
+Prerequisites
+-------------
+
+Before you can query MongoDB from your .NET application, you must set up
```
Comment (dacharyc) [inline source/sdk/react-native/app-services/query-mongodb.txt:49]:
```suggestion
Before you can query MongoDB from your React Native application, you must set up
```

**Review by dacharyc — APPROVED** [review]
Couple of corrections, and we should probably link out to the Java SDK's MongoClient if we're doing that in all the other SDKs, but overall LGTM.

File: source/web/mongodb.txt [hunk 5]
```diff
@@ -11,11 +11,26 @@ Query MongoDB - Web SDK
    :depth: 2
    :class: singlecol
 
+You can query data stored in MongoDB Atlas directly from your client application code
+by using the Realm Web SDK's :js-sdk:`MongoDB client <TRealm-MongoDB.html>`
```
Comment (mongodben) [inline source/web/mongodb.txt:15]:
```suggestion
by using the Realm Web SDK's :js-sdk:`MongoDB client <Realm-MongoDB.html>`
```

File: source/sdk/node/examples/query-mongodb.txt [hunk 6]
```diff
@@ -12,6 +12,13 @@ Query MongoDB - Node.js SDK
    :depth: 2
    :class: singlecol
 
+You can query data stored in MongoDB Atlas directly from your client application code
+by using the Realm Node.js SDK's :js-sdk:`MongoDB client <TRealm-MongoDB.html>`
```
Comment (mongodben) [inline source/sdk/node/examples/query-mongodb.txt:16]:
```suggestion
by using the Realm Node.js SDK's :js-sdk:`MongoDB client <Realm-MongoDB.html>`
```

File: source/sdk/react-native/app-services/query-mongodb.txt [hunk 7]
```diff
@@ -12,6 +12,13 @@ Query MongoDB - React Native SDK
    :depth: 2
    :class: singlecol
 
+You can query data stored in MongoDB Atlas directly from your client application code
+by using the Realm React Native SDK's :js-sdk:`MongoDB client <TRealm-MongoDB.html>`
```
Comment (mongodben) [inline source/sdk/react-native/app-services/query-mongodb.txt:16]:
```suggestion
by using the Realm React Native SDK's :js-sdk:`MongoDB client <Realm-MongoDB.html>`
```

**Pull request merged**